test/Routes.test.js

```diff
@@ -101,12 +101,16 @@ describe('Routes', () => {
       req = request(server.app);
     });
 
-    after(helper.close);
+    afterAll(helper.close);
 
     it('GET request with headers as an array', (done) => {
+      const expected =
+        process.version.split('.')[0] === 'v8'
+          ? 'key1=value1,key2=value2'
+          : 'key1=value1, key2=value2';
```

Comment:
Maybe just remove spaces from expected? I am afraid another version can also have same problem?

Reply:
| version | output |space|
| :--- | :--- |:--- |
| 6| `key1=value1, key2=value2` |y|
| 7 | `key1=value1,key2=value2` |n| 
| 8| `key1=value1,key2=value2` |n|
| 9 | `key1=value1,key2=value2` |n| 
| 10| `key1=value1, key2=value2` |y|
| 11 | `key1=value1, key2=value2` |y|

6, 10, 11 have the spaces but 7, 8, 9 don't have.

So, I change the code to `['v7', 'v8', 'v9'].includes`.

Reply:
@hiroppy Can you send this change in commit?

Reply:
added to https://github.com/webpack/webpack-dev-server/pull/1650/files#diff-f1603d639761fe762d975b5ce05e20b1R107

Reply:
Separated this commit(https://github.com/webpack/webpack-dev-server/pull/1650/commits/10e4bc27c96b6c70b8020cf2e034796a5b552fc0) /cc @evilebottnawi